Transformer/TransformationLanguage.md

```diff
@@ -0,0 +1,125 @@
+<!-- TOC depthFrom:1 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Introduction](#introduction)
+- [JSONata Overview](#jsonata-overview)
+	- [Exemplary Functions](#exemplary-functions)
+		- [String Functions](#string-functions)
+		- [Numeric Functions](#numeric-functions)
+		- [Array Functions](#array-functions)
+- [Exemplary JSONata Transformation](#exemplary-jsonata-transformation)
+
+<!-- /TOC -->
+# Introduction
+As described in [README.md](/REAMDE.md) a transformer semantically transforms one JSON object into another. For this purpose a programming language or a transformation language such as JSONata can be used. <br>
+
+Short description of model transformation languages in general:
+
+> A model transformation, which is essentially a program which operates on models, can be written in a general-purpose programming language, such as Java. However, special-purpose model transformation languages can offer advantages, such as syntax that makes it easy to refer to model elements.
+
+_Source:_ [Wikipedia](https://en.wikipedia.org/wiki/Model_transformation_language)
+
+Due to its focus on JSON objects and its simplicity, JSONata is the recommended transformation language for creating a transformer.
+
+# JSONata Overview
+- Is a Lightweight query and transformation language for JSON data
+- Helps to extract specific meaningful data from JSON data using a simple logic
+- Supports more than 20 operators
+- Includes numerous functions such as string-, numeric-, array- and object-functions
+
+## Exemplary Functions
```

Comment:
More examples can be seen at https://support.elastic.io/support/solutions/articles/14000069448-jsonata-powered-mapper

Reply:
Good point. I'll have a look if I can incorporate it.